nodejs_install.sh

```diff
@@ -21,6 +21,7 @@ else
   echo -e "\n==== Installing NodeJS && NPM && Update Browserlist ====\n"
   sudo apt install -y nodejs  
   sudo apt install -y npm
+  sudo npx update-browserslist-db@latest
```

Comment:
This should be in your package.json.

Reply:
It is, I brought this file back from my other crown app repo when I lost it yesterday.